### PR TITLE
[comp-eval] UI>content>errata tab, Pagination testing

### PR DESCRIPTION
**Pull request by damoore044**
### Update Errata UI Pagination testing
- to standards for 6.15.0 for setup w/ global registration, and new host UI views
- depends on [airgun#1152](https://github.com/SatelliteQE/airgun/pull/1152)

### Implement
- use of registered host fixture, setup w/ global registration, subscribed to content.
- method to publish and promote cv, wait for any task(s).
- migrate Airgun UI view; from `legacy contenthost UI > errata page` ,
   to `new host UI > content > errata tab` .

### Handle Failure: expected updated pagination, but remains the same:
- Sometimes, `applicable_errata_count` for the host has already updated after some install, but pagination on the table remains the same as before. I used `wait_for` to give the `new host>content>errata tab` time to reload with updated entries /total_items/total_pages etc. Timeout set for 30s on the tab loading new table and pagination, in case satellite is slow.

### Affected tests and PRT case:
- `test_content_host_errata_page_pagination` renamed to `test_host_content_errata_tab_pagination`
- `test_end_to_end`  by modifying fixture `registered_contenthost`
```
trigger: test-robottelo
pytest: tests/foreman/ui/test_errata.py -k 'test_host_content_errata_tab_pagination or test_end_to_end'
airgun: 1152
```

### CODE REVIEW

**Comment by damoore044** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_errata.py -k 'test_host_content_errata_tab_pagination or test_end_to_end'
airgun: 1152

**Comment by damoore044** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_errata.py -k 'test_host_content_errata_tab_pagination'
airgun: 1152

**Comment by damoore044** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_errata.py -k 'test_host_content_errata_tab_pagination'
airgun: 1152

**Comment by damoore044** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_errata.py -k 'test_host_content_errata_tab_pagination'
airgun: 1152

**Comment by vijaysawant** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_errata.py -k 'test_host_content_errata_tab_pagination'
airgun: 1152

**Comment by damoore044** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_errata.py -k 'test_host_content_errata_tab_pagination'
airgun: 1152

**Comment by vsedmik** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_errata.py -k 'test_host_content_errata_tab_pagination'
airgun: 1152

**Comment by damoore044** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_errata.py -k 'test_host_content_errata_tab_pagination'
airgun: 1152

**Comment by damoore044** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_errata.py -k 'test_host_content_errata_tab_pagination'
airgun: 1152

**Comment by damoore044** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_errata.py -k 'test_host_content_errata_tab_pagination'
airgun: 1152

**Review by JacobCallahan — CHANGES_REQUESTED** [review]
one minor request. well done

File: tests/foreman/ui/test_errata.py [hunk 1]
```diff
@@ -395,88 +481,239 @@ def test_end_to_end(
...
+    )
+    module_cv = module_cv.read()
+    cv_publish_promote(
+        module_target_sat,
+        org,
+        module_cv,
+        module_lce,
+    )
+    registered_contenthost.add_rex_key(satellite=module_target_sat)
+    assert registered_contenthost.subscribed
```
Comment (vijaysawant) [inline tests/foreman/ui/test_errata.py:568]: This is not require, as subscription status already asserted inside `registered_contenthost` fixture after registration of content host.
```suggestion
```

**Review by vijaysawant — CHANGES_REQUESTED** [review]
Request to remove assersion condition from test module, because it is already handled inside fixture

**Comment by damoore044** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_errata.py -k 'test_host_content_errata_tab_pagination or test_end_to_end'
airgun: 1152

**Comment by damoore044** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_errata.py -k 'test_host_content_errata_tab_pagination or test_end_to_end'
airgun: 1152

**Review by JacobCallahan — APPROVED** [review]

**Pull request merged**